<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>702</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Logo.jpg</normaloff>../Logo.jpg</iconset>
  </property>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>이미지 불러오기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>이미지 저장하기</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>600</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>컬러 바꾸기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hue">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>채도 바꾸기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sat">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>밝기 바꾸기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="val">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="colorSelectButton">
      <property name="text">
       <string>수정할 컬러 선택</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="colorFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>1031</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="restoreButton">
      <property name="text">
       <string>열화 손상 복원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pretreatmentButton">
      <property name="text">
       <string>이미지 전처리</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="coloringButton">
      <property name="text">
       <string>딥러닝 채색</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>639</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>색 범위 지정</string>
   </property>
  </widget>
  <widget class="QSlider" name="setRange">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>630</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>AddButton</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RemoveButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>650</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RemoveButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>599</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>브러쉬 사이즈</string>
   </property>
  </widget>
  <widget class="QSlider" name="setBrushSize">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>600</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>imgLabel</zorder>
  <zorder>resultLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>label_13</zorder>
  <zorder>setRange</zorder>
  <zorder>AddButton</zorder>
  <zorder>RemoveButton</zorder>
  <zorder>label_14</zorder>
  <zorder>setBrushSize</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
